cfme/utils/trackerbot.py

```diff
@@ -1,16 +1,19 @@
 import argparse
 import json
 import re
+import time
+import urllib
 import urlparse
 from collections import defaultdict, namedtuple
-from datetime import date
-import urllib
+from datetime import date, datetime
 
+import attr
 import slumber
 import requests
-import time
+from bs4 import BeautifulSoup
```

Comment:
**QUESTION**
I don't see `bs4` package in `frozen.txt`. Should it be added? If yes, please consider to use `lxml` instead.

Reply:
Odd, I don't recall actually installing it in my venv, but I have it? Its actually in `pip freeze` as `beautifulsoup4==4.6.0`, but is imported through bs4.  Will evaluate lxml and update either frozen.txt or my implementation.